clickhouse/db-system-events.yml

```diff
@@ -122,12 +122,17 @@ observation:
         label: Uncompressed bytes merged
         unit: bytes
         description: Uncompressed bytes that was read for background merges. This is the number before merge.
+      - name: merge.count
```

Comment:
So effectively we will switch "Running merges" metric in CH Merges chart into "Merge count" metric (for anyone that upgrades the agent). Would it make sense to add "Running merges" back to the same chart?

Reply:
No. Currently, the chart has Running merges and merge time (which is a counter). This change will replace Running merges to merge count.  So after this change, we won't show running merges anywhere in the reports. I was thinking about adding it background tasks charts ( which show currently running background tasks )

Reply:
Hm, that is exactly what I said :).